Show details of parents in output of ``paasta status``

Rob-Johnson opened the pull request:
<img width="902" alt="screen shot 2016-01-28 at 16 17 39" src="https://cloud.githubusercontent.com/assets/1389671/12650428/b91d65d2-c5da-11e5-851a-93353453882d.png">

This is slightly wasteful, in that we're doing the lookups for the parent jobs multiple times. I'm ok with that for now, but probably a case where we can optimize if we get to the point where it's impacting on performance.

Closes #96

solarkennedy commented on paasta_tools/chronos_serviceinit.py:
```diff
@@ -167,6 +190,7 @@ def format_chronos_job_status(job, running_tasks, verbose):
     disabled_state = _format_disabled_status(job)
     (last_result, formatted_time) = _format_last_result(job)
     schedule = _format_schedule(job)
+    parents = _format_parents(job)
```
Although probably not the simplest to do with how the code is now. Can you make this only show up in verbose mode?

As we use chronos more, paasta status is going to start getting really long. Most of the time people won't be interested in this data, and if they *are* inspecting a particular instance, *then* I think this detailed parent status will be useful.

solarkennedy commented:
Can you also merge master to simplify and get test fixes?

Rob-Johnson commented:
Ok, I've updated. If anyone has any better ideas for the wording on non-verbose dependent jobs, then I'm open to suggestions. 

updated screenshots:

non-verbose dependent job:
![screen shot 2016-02-01 at 16 21 25](https://cloud.githubusercontent.com/assets/1389671/12723267/9fe31444-c900-11e5-8278-b46abcb045ff.png)

verbose dependent job:
![screen shot 2016-02-01 at 16 25 01](https://cloud.githubusercontent.com/assets/1389671/12723279/a914e63c-c900-11e5-9a6f-c2ed2bd354e8.png)

scheduled job:
![screen shot 2016-02-01 at 16 26 14](https://cloud.githubusercontent.com/assets/1389671/12723282/b08d8996-c900-11e5-9429-e6de59dec9c6.png)

solarkennedy commented:
How about making the non-verbose option a bit more non-verbose:

```
Instance: examplejob.foo
  Status:  Scheduled
  Last:    OK (last run time)
  Parents: foo, bar
  Command:
  Mesos: 
```

solarkennedy commented:
In other words, go ahead and say the parents if you know there are some, and just have it replace the schedule part. (hopefully that is easy)

Rob-Johnson commented:
ok, final run:

![screen shot 2016-02-03 at 14 44 52](https://cloud.githubusercontent.com/assets/1389671/12785641/54909652-ca85-11e5-9a8a-e6d83a350c71.png)
![screen shot 2016-02-03 at 14 44 59](https://cloud.githubusercontent.com/assets/1389671/12785640/548f153e-ca85-11e5-911b-8f125c88266b.png)
![screen shot 2016-02-03 at 14 45 05](https://cloud.githubusercontent.com/assets/1389671/12785642/549436cc-ca85-11e5-89d9-5ce6c7202353.png)

giuliano108 commented on paasta_tools/chronos_tools.py:
```diff
@@ -177,14 +182,6 @@ def __init__(self, service, instance, cluster, config_dict, branch_dict):
             branch_dict=branch_dict,
         )
 
-    def __eq__(self, other):
```
Is this officially gone?

Rob-Johnson replied:
ah, great catch.

Rob-Johnson commented:
@giuliano108 corrected

giuliano108 commented on paasta_tools/chronos_serviceinit.py:
```diff
@@ -131,12 +131,80 @@ def _format_last_result(job):
+    elif job_type == chronos_tools.JobType.Scheduled:
+        return 'Schedule'
+    else:
+        return 'Unknown'
```
I was expecting this `raise`...

solarkennedy commented:
ship
